views/confirm-company.html

```diff
@@ -0,0 +1,133 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Confirm the company
+{% endblock %}
+{% block signOut %}{% endblock %}
```

Comment:
we can remove the sign out block. We removed it from the layout so we don't need to override it in this file.